docs/pages/server-access/troubleshooting-server.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: Troubleshooting Server Access
+description: Describes common issues and solutions for access to servers.
+---
+
+This section describes common issues that you might encounter in managing access to servers
+with Teleport and how to work around or resolve them.
+
+## Starting SSH sessions fails
+
+By default, if you use SSH to connect to a server, Teleport runs as the user login that starts 
+the SSH session. If starting the session fails, it might be due to the Teleport file permission
+settings for the user login that Teleport is using to connect to the remote servr.
+This issue might be caused by systems that prevent newly installed software from being executable 
+by any user.  
+
+### Symptom
+
+This issue results in a disconnected session and an error message similar to the following:
+
+```text
+Failed to launch: fork/exec /proc/self/exe: permission denied.
+Process exited with status 255
+```
+
+### Solution
+
+You should check the permission settings for the `teleport` service. 
+
+To check the permissions on the teleport service:
+
+1. Open a terminal shell on the computer where you have installed the `teleport` service.
+
+1. Determine the location of the Teleport binary by running the following command:
+   
+   ```code
+   which teleport
+   ```
+
+1. Check the permission settings for the binary by running the following command:
+   
+   ```code
+   ls -al <Var name="/usr/local/bin/teleport" />
+   ```
+
+   The command should return output similar to the following:
+   
+   ```text
+   -rwxr-xr-x  1 root  wheel  531849504 Aug 30 18:32 /usr/local/bin/teleport
+   ```
+   
+   If you don't see the permission that allows other users to read and execute (-rwxr-x**r-x**), 
+   you should update the permissions. For example:
+
+   ```code
+   chmod o+rx teleport
+   ```
+   
+1. Restart the `teleport` service.
+
+## Missing logins for single sign-on users
+
+If you use an external identity provider to enable single sign-on for users, you should be sure
+to assign logins for those users.
+
+### Symptom
+
+Users who have access to Teleport through an authentication connector for an external identity
+provider don't see any of the logins they need to access remote resources.
+
+### Solution
+
+To fix this issue, you can assign logins to your single sign-on users or modify the traits in 
+the Teleport roles assigned to users through their group membership in the external identity provider.
```

Comment:
I would recommend linking to another docs page here so users can find out more about Teleport traits and role mapping.